theano/misc/tests/test_gnumpy_utils.py

```diff
@@ -44,9 +40,6 @@ def test2(shape=(3, 4, 5)):
     Make sure that the gnumpy conversion is exact from CudaNdarray to
     garray back to CudaNdarray.
     """
-    gpu = theano.sandbox.cuda.basic_ops.gpu_from_host
```

Comment:
The test might be checking if the theano.function can be compiled. Just remove the final assignment please (i.e. call `theano.function` but don't assign it to `ii`)